Source: Alexander Scheuermann <alex+exherbo@tnix.de>
Upstream: No
Reason: Add default daemon configuration
--
--- a/prosody.cfg.lua.dist      2022-06-07 15:13:11.489020011 +0200
+++ b/prosody.cfg.lua.dist      2022-06-07 15:14:48.644629188 +0200
@@ -183,10 +183,11 @@
 -- Logging configuration
 -- For advanced logging see https://prosody.im/doc/logging
 log = {
-	info = "prosody.log"; -- Change 'info' to 'debug' for verbose logging
-	error = "prosody.err";
+	-- info = "prosody.log"; -- Change 'info' to 'debug' for verbose logging
+	-- error = "prosody.err";
 	-- "*syslog"; -- Uncomment this for logging to syslog
 	-- "*console"; -- Log to the console, useful for debugging when running in the foreground
+	info = "*syslog";
 }
 
 
@@ -205,6 +206,13 @@
 -- Location of directory to find certificates in (relative to main config file):
 certificates = "certs"
 
+-- mod_posix configuration
+-- Provide daemon settings required by the systemd service file.
+run_as_root = false
+daemonize = true
+pidfile = "/run/prosody/prosody.pid"
+umask = "027"
+
 ----------- Virtual hosts -----------
 -- You need to add a VirtualHost entry for each domain you wish Prosody to serve.
 -- Settings under each VirtualHost entry apply *only* to that host.
